<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>431</width>
      <height>264</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="pFNameLabel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>PFName</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="pFNameLineEdit">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="pLNameLabel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>PLName</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="pLNameLineEdit">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pDoBLabel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>PDoB</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="pDoBLineEdit">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="pGenderLabel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>PGender</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="pPhoneLabel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>PPhone</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="pPhoneLineEdit">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="pAddressLabel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>PAddress</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="pAddressLineEdit">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="pushButtonInsert">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="pGenderLineEdit">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
